src/main/java/org/opentripplanner/graph_builder/GraphBuilder.java

```diff
@@ -126,10 +127,18 @@ public static GraphBuilder create(
             osmModule.banDiscouragedWalking = config.banDiscouragedWalking;
             osmModule.banDiscouragedBiking = config.banDiscouragedBiking;
             graphBuilder.addModule(osmModule);
-            PruneFloatingIslands pruneFloatingIslands = new PruneFloatingIslands();
-            pruneFloatingIslands.setPruningThresholdIslandWithoutStops(config.pruningThresholdIslandWithoutStops);
-            pruneFloatingIslands.setPruningThresholdIslandWithStops(config.pruningThresholdIslandWithStops);
-            graphBuilder.addModule(pruneFloatingIslands);
+	    if (OTPFeature.PruneIslands.isOn()) { // original pruning
```

Comment:
Also, there are a few places where the indentation is off.

Reply:
The issue seems to be that the new lines are indented using tabs, while the old code uses spaces

Reply:
Tabs should not be used. We base our style-guide on the Google style:

https://google.github.io/styleguide/javaguide.html#s2.3.1-whitespace-characters

Reply:
I re-checked the changed files and removed some tabs. I did not convert all tabs in the existing code base though.

Reply:
I made the code-style none optional in this PR #3636. The auto-format is still flaky, so it still need work - but at least that is no need to import it, when there is changes. When we are satisfied with the auto-import we will reformat the entire code base - some time after the release of OTP 2.1.